src/api/app/controllers/webui/staging/workflows_controller.rb

```diff
@@ -56,14 +59,38 @@ def destroy
     @staging_workflow.staging_projects.where(id: params[:staging_project_ids]).destroy_all
 
     if @staging_workflow.destroy
-      flash[:success] = "Staging Workflow for #{@project.name} was successfully deleted."
+      flash[:success] = "Staging for #{@project.name} was successfully deleted."
       render js: "window.location='#{project_show_path(@project)}'"
     else
-      flash[:error] = "Staging Workflow for #{@project.name} couldn't be deleted: #{@staging_workflow.errors.full_messages.to_sentence}."
+      flash[:error] = "Staging for #{@project.name} couldn't be deleted: #{@staging_workflow.errors.full_messages.to_sentence}."
       render js: "window.location='#{staging_workflow_path(@staging_workflow)}'"
     end
   end
 
+  def assign_managers_group
```

Comment:
What about putting this in a dedicated controller as update action how we do already for most other controllers?

```
Staging::Project::GroupsController#update
```

Reply:
But this is updating the StagingWorkflow, not the Group.

Reply:
Then what about 

```
Staging::Project::ManagersController#update
```

Reply:
Still we are updating the StagingWorkflow, not the Managers group

Reply:
Then 

``
Staging::Workflow::ManagersController#update
``

C'mon, either say you don't want to do it or come up with some other idea ...

Reply:
I won't do it :heart:

Reply:
`Staging::Workflow::AssignManagersController#create` ??

Reply:
Nevermind, I will rename the `assign_managers_group` action to `update` and let it like it is... in the end is just updating one part of the `StagingWorkflow`

Reply:
Ok, that's also a solution and we can extend in the future.